app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::SessionsController < ApplicationController
+  before_action :authorize_access_request!, only: [:destroy]
+
+  def new
+    user = User.new(user_params)
+    if user.save
+      tokens = LoginHandler.session(build_payload(user))
+      set_response(tokens)
+      render json: { csrf: tokens[:csrf], user: UserSerializer.new(user) }, status: :created
+    else
+      render json: { error: user.errors.full_messages.join(' ') }, status: :unprocessable_entity
```

Comment:
forgot that rails has this built in, I probably don't need many of the error methods in the error class. This makes it redundant